locales/base/translation.json

```diff
@@ -1990,10 +1990,11 @@
     "subtitle": "What would you like to do first?",
     "later": "I'll explore later",
     "options": {
-      "add": "Add funds",
+      "add": "Add funds to your wallet",
       "dapp": "Find ways to use crypto",
-      "profile": "Build my profile",
-      "learn": "Learn about Celo"
+      "earn": "Explore earning opportunities",
+      "profile": "Build your profile",
+      "learn": "Learn about {{appName}} Points"
```

Comment:
we should create a new key for this right? the "learn" key is used for the celo education navigation in older app versions

Reply:
My bad, will fix.

Reply:
@kathaypacific Should this involve replacing [AdventureCardName.Learn](https://github.com/valora-inc/wallet/blob/main/src/onboarding/types.ts#L5) with smth like `AdventureCardName.LearnPoints`? 
I'm still not fully in the context of the whole analytics flow so I'm not sure if change like this is required.

Reply:
yes i think you can change it to Points or LearnPoints, and remove the enum members that are no longer used :) the analytics property is just to allow us to understand which options are being selected, so it's expected that some of the enum values will no longer appear in analytics